src/ui/buttons/buttonDesigns/alternateSlideLogoDesign.jsx

```diff
@@ -0,0 +1,127 @@
+/* @flow */
+/** @jsx node */
+import { LOGO_CLASS } from '@paypal/sdk-logos/src';
+import { node, Fragment, type ChildType } from 'jsx-pragmatic/src';
+
+import { CLASS } from '../../../constants';
+import { BUTTON_SIZE_STYLE } from '../config';
+
+import type {  ContentOptions } from './types';
+
+type AnimationProps = {|
+    paypalLabelContainerElement : Object,
+    paypalLogoStartingLeftPosition : string
+|};
+
+
+export const FADEOUT_LOGO_SHOW_LABEL_TEXT_CONFIG = {
+    min:                            BUTTON_SIZE_STYLE.tiny.minWidth,
+    max:                            BUTTON_SIZE_STYLE.medium.maxWidth,
+    cssClasses: {
+        DOM_READY:                  CLASS.DOM_READY,
+        ANIMATION_CONTAINER:        'personalized-design-container',
+        PAYPAL_LOGO:                LOGO_CLASS.LOGO,
+        ANIMATION_LABEL_CONTAINER:  'personalized-label-container',
+        PAYPAL_BUTTON_LABEL:        CLASS.BUTTON_LABEL
+    }
+};
+
+export function AlternateSlideLogoComponent({ designLabelText } : ContentOptions) : ChildType {
+    const CONTAINER_CLASS = FADEOUT_LOGO_SHOW_LABEL_TEXT_CONFIG.cssClasses.ANIMATION_CONTAINER;
+    const LABEL_CLASS = FADEOUT_LOGO_SHOW_LABEL_TEXT_CONFIG.cssClasses.ANIMATION_LABEL_CONTAINER;
+   
+    return (
+        <Fragment>
+            <div class={ LABEL_CLASS } data-design-experiment='104530'> <span>{designLabelText}</span></div>
+            <style innerHTML={ `
+                .${ CLASS.DOM_READY } .${ CONTAINER_CLASS } img.${ LOGO_CLASS.LOGO }{
+                    position: relative;
+                }
+                
+                .${ CONTAINER_CLASS } .${ LABEL_CLASS } {
+                    position: fixed;
+                    opacity: 0; 
+                    color: #142C8E;
+                    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+                    font-size: 10px;
+                }
+            ` } />;
+        </Fragment>
+    );
+}
+
+// Returns label container if the button sizes match
+export const getAlternateSlideLogoProps = function(document : Object, configuration : Object) : AnimationProps | null {
+    const { ANIMATION_CONTAINER, ANIMATION_LABEL_CONTAINER, PAYPAL_BUTTON_LABEL, PAYPAL_LOGO } = configuration.cssClasses;
+    const { min, max } = configuration;
+    // get the animation main container to force specificity( in css ) and make sure we are running the right animation
+    const animationContainer = (document && document.querySelector(`.${ ANIMATION_CONTAINER }`)) || null;
+    if (!animationContainer) {
+        return null;
+    }
+
+    // return null if animation should not be played for the button size
+    const animationContainerWidth = animationContainer.offsetWidth;
+    if (animationContainerWidth < min || animationContainerWidth > max) {
+        // remove label element from dom
+        animationContainer.querySelector(`.${ ANIMATION_LABEL_CONTAINER }`).remove();
```

Comment:
If this doesn't break stuff please add similar logic in other designs

Reply:
what do you mean, if this doesn't break stuff? this code only remove the label component if no animation is gonna be run but the label component was rendered already.

Reply:
Yeah, so it theoretically shouldn't break stuff. But if you test it and it looks fine can you add it

Reply:
yeah I have tested and it works fine. You can see when the button has 306px the label element does not exist because gets removed by this line of code
![Screen Shot 2021-12-02 at 3 31 11 PM](https://user-images.githubusercontent.com/7244087/144498439-323bfb47-c40d-4880-a53b-9bfd7d4d07de.png)
:
I normally insert the element inside the element with class `paypal-button-label-container`